index.js

```diff
@@ -0,0 +1,53 @@
+let input = document.getElementById('operation-input');
+
+let zero = document.getElementById('zero');
+let one = document.getElementById('one');
+let two = document.getElementById('two');
+let three = document.getElementById('three');
+let four = document.getElementById('four');
+let five = document.getElementById('five');
+let six = document.getElementById('six');
+let seven = document.getElementById('seven');
+let eight = document.getElementById('eight');
+let nine = document.getElementById('nine');
+let dot = document.getElementById('dot');
+
+let leftparenthesis = document.getElementById('leftparenthesis')
+let rightparenthesis = document.getElementById('rightparenthesis')
+let rest = document.getElementById('rest')
+let clear = document.getElementById('allclear')
+let division = document.getElementById('division')
+let multiplication = document.getElementById('multiplication')
+let minus = document.getElementById('minus')
+let equals = document.getElementById('equals')
+let plus = document.getElementById('plus')
+
+zero.addEventListener('click', ()=>updateOperation('0'));
```

Comment:
You probably could just reduce the code for this:

`document.getElementById('zero').addEventListener('click', ()=>updateOperation('0'));`

Reply:
Just if you will not use any more the button

Reply:
yy